packages/cli-command/src/command.js

```diff
@@ -62,8 +62,8 @@ function exit(exitCode, reason = '') {
 // and other common command helpers and properties.
 async function runCommandWithContext(parsed) {
   let { command, flags, args, argv, log } = parsed;
-  // include flags, args, argv, logger, exit helper, and env info
-  let context = { flags, args, argv, log, exit };
+  // include command as parentCommand, flags, args, argv, logger, exit helper, and env info
+  let context = { parentCommand: command, flags, args, argv, log, exit };
```

Comment:
you can just call it `command`. no need to rename it to parentCommand. Or if you want to differentiate between user command [argv] and this then call it cliCommand